js/stdlib/ts/interfaces.ts

```diff
@@ -161,7 +162,7 @@ export interface Stdlib_User<Provider, ProviderEnv, ProviderName, Token, Contrac
   balanceOf: (acc: Account, token?: Token) => Promise<BigNumber>
   balancesOf: (acc: Account, tokens: Array<Token | null>) => Promise<Array<BigNumber>>
   minimumBalanceOf: (acc:Account) => Promise<BigNumber>
-  transfer: (from: Account, to: Account, val?: BigNumber, token?: Token) => Promise<unknown>
+  transfer: (from: Account, to: Account, amount: any, token?: Token, opts?: TransferOpts) => Promise<unknown>
```

Comment:
`to` should be `Address`

Reply:
i didn't change this; i can fix it in another pr along with the other ones